ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -0,0 +1,383 @@
+    )
+    if subscription_info:
+        logger.info(f"The Subscription is created successfully")
+    else:
+        logger.error(f"The subscription is not installed properly")
+    return subscription_info
+
+
+def create_instance_in_clusterlogging(sc_name=None):
+    """
+    Creation of instance for clusterlogging that creates PVCs,
+    ElasticSearch, curator fluentd and kibana pods and checks for all
+    the pods and PVCs
+
+    Args:
+        sc_name (str) : Storage class name to create PVCs
+
+    Returns:
+        logging_instance (dict): Contains all detailed information of the
+        instance such as pods that got created, its resources and limits
+        values, storage class and size details etc.
+
+    """
+    inst_data = templating.load_yaml_to_dict(constants.CL_INSTANCE_YAML)
+    inst_data['spec']['logStore']['elasticsearch']['storage']['storageClassName'] = sc_name
+    inst_data['spec']['logStore']['elasticsearch']['storage']['size'] = "200Gi"
+    helpers.create_resource(wait=False, **inst_data)
+    oc = ocp.OCP('v1', 'ClusterLogging', 'openshift-logging')
+    logging_instance = oc.get(resource_name='instance', out_yaml_format='True')
+    if logging_instance:
+        logger.info("Successfully created instance for cluster-logging")
+        logger.debug(logging_instance)
+    else:
+        logger.error("Instance for clusterlogging is not created properly")
+
+    pv_obj = ocp.OCP(
+        kind=constants.POD, namespace='openshift-logging'
+    )
+    ret = pv_obj.wait_for_resource(condition=constants.STATUS_RUNNING,
+                                   resource_count=11,
+                                   timeout=150, sleep=5
+                                   )
+    assert ret, "Pods are not in Running state."
+    logger.info("All pods are in Running state")
+    pv_obj = ocp.OCP(
+        kind=constants.PVC, namespace='openshift-logging'
+    )
+    ret = pv_obj.wait_for_resource(condition=constants.STATUS_BOUND,
+                                   resource_count=3, timeout=150, sleep=5
+                                   )
+    assert ret, "PVCs are not in bound state."
+    logger.info("PVCs are Bound")
+    return logging_instance
+
+
+@retry(CommandFailed, 5, 30, 2)
+def check_health_of_clusterlogging():
+    """
+    * Checks for ElasticSearch, curator, fluentd and kibana pods in
+    openshift-logging namespace
+
+    * And check for the health of cluster logging, If status is green then the
+    cluster is healthy,if status is red then health is bad
+
+    Returns:
+        pod_list (list) : Gives all the pods that are present in the namespace
+
+    """
+
+    pod_list = []
+    pods = get_all_pods(namespace='openshift-logging')
+    logger.info("Pods that are created by the instance")
+    for pod in pods:
+        pod_list.append(pod.name)
+    logger.info(pod_list)
+    elasticsearch_pod = [i for i in pod_list if i.startswith('elasticsearch')]
```

Comment:
Let's try to avoid ambiguous var names like `i` whenever possible:
```suggestion
    elasticsearch_pod = [
        pod for pod in pod_list if pod.startswith('elasticsearch')
    ]
```